Error 2, syntax error: unexpected EOF when using 'make template_init'

## Current behaviour
I tried running make template_init from the [getting started guide](https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/master/GETTING_STARTED/getting_started_modules.md). 

When running `make template_init` I get following error:

```
`\e[91m
Warning! This will clean your template. Do you want to continue? [y/n]\e[0m
y
if [ "$answer" != "y" ]; then echo "Aborting!" && exit; fi
Aborting!
Starting to clean your template!
\e[0m
for folder in "template_example" "example/vagrant_box_example" "CHANGELOG.md"
/bin/sh: -c: line 1: syntax error: unexpected end of file
make: *** [template_init] Error 2 `
```

## Expected behaviour
Should clean template, i.e., delete redundant folders and move old README etc.

## How to reproduce?
Fork repo, clone repo then run command `make template_init`.
Using macOS

## Suggestion(s)/solution(s) [Optional]

## Checklist (after created issue)
- [x] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

@pdmthorsrud

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,7 +6,7 @@
 - Moved colour definitions of ‘template_init’ recipe into variables. [no issue]
 
 ### Fixed
-- bug where 'make template_init' would crash on error #2 EOF.
+- bug where 'make template_init' would crash on End of File error. [Issue #92]
 
 ## [0.1.0 UNRELEASED]
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,7 +6,7 @@
 - Moved colour definitions of ‘template_init’ recipe into variables. [no issue]
 
 ### Fixed
-- bug where 'make template_inti' would crash on error #2 EOF.
+- bug where 'make template_init' would crash on error #2 EOF.
 
 ## [0.1.0 UNRELEASED]
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,11 +2,8 @@
 
 ## [0.1.1 UNRELEASED]
 
-### Added
-- Colour Scheme as ad hoc variables instead of inline ad lib. 
-
 ### Changed
-- Colours in 'make template_init' termplate text output 
+- Moved colour definitions of ‘template_init’ recipe into variables. [no issue]
 
 ### Fixed
 - bug where 'make template_inti' would crash on error #2 EOF.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,15 @@
 # Changelog
 
+## [0.1.1 UNRELEASED]
+
+### Added
+- Colour Scheme as ad hoc variables instead of inline ad lib. 
+
+### Changed
+- Colours in 'make template_init' termplate text output 
+
+### Fixed
+- bug where 'make template_inti' would crash on error #2 EOF.
 ## [0.1.0 UNRELEASED]
 
 ### Added

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,7 +6,7 @@
 - Moved colour definitions of ‘template_init’ recipe into variables. [no issue]
 
 ### Fixed
-- bug where 'make template_init' would crash on End of File error. [Issue #92]
+- bug where 'make template_init' would crash on End of File error. [#92](https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/92)
 
 ## [0.1.0 UNRELEASED]
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,7 @@
 
 ### Fixed
 - bug where 'make template_inti' would crash on error #2 EOF.
+
 ## [0.1.0 UNRELEASED]
 
 ### Added

--- a/Makefile
+++ b/Makefile
@@ -121,7 +121,7 @@ template_init:
 	done
 
 	@echo "\nMoving README.md to .github/template_specific as old_README.md" 
-	@mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
+	mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED"
 	@mv GETTING_STARTED .github/template_specific/ && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"

--- a/Makefile
+++ b/Makefile
@@ -120,16 +120,16 @@ template_init:
 		rm -rf $$folder && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}" ; \
 	done
 
-	@echo "\nMoving README.md to .github/template_specific as old_README.md" 
+	@echo -n "\nMoving README.md to .github/template_specific as old_README.md" 
 	mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED"
+	@echo -n "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED"
 	mv GETTING_STARTED .github/template_specific/ && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo "\nCreating a clean README.md"
+	@echo -n "\nCreating a clean README.md"
 	cat .github/template_specific/README_template.md >> README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo "Creating a clean CHANGELOG.md "
+	@echo -n "Creating a clean CHANGELOG.md "
 	echo "# Changelog\n\n## 0.0.1 [UNRELEASED]\n\n###Added\n\n###Changed\n\n###Fixed\n" >> CHANGELOG.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "${BLUE}\nDone! You are all set to start developing!${RESET}"

--- a/Makefile
+++ b/Makefile
@@ -124,7 +124,7 @@ template_init:
 	mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED"
-	@mv GETTING_STARTED .github/template_specific/ && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
+	mv GETTING_STARTED .github/template_specific/ && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "\nCreating a clean README.md"
 	cat .github/template_specific/README_template.md >> README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"

--- a/Makefile
+++ b/Makefile
@@ -120,7 +120,7 @@ template_init:
 		rm -rf $$folder && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}" ; \
 	done
 
-	@echo -n "\nMoving README.md to .github/template_specific as old_README.md" 
+	@echo -n "\nMoving README.md to .github/template_specific as old_README.md " 
 	mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo -n "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED"

--- a/Makefile
+++ b/Makefile
@@ -130,7 +130,7 @@ template_init:
 	cat .github/template_specific/README_template.md >> README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "Creating a clean CHANGELOG.md "
-	echo "# Changelog\n\n## 0.0.1 [UNNRELEASED]\n\n###Added\n\n###Changed\n\n###Fixed\n" >> CHANGELOG.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
+	echo "# Changelog\n\n## 0.0.1 [UNRELEASED]\n\n###Added\n\n###Changed\n\n###Fixed\n" >> CHANGELOG.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "${BLUE}\nDone! You are all set to start developing!${RESET}"
 	@echo "${YELLOW}\nPS! If you want to keep a deleted folder, you can undo by running:\n  git reset HEAD <file/folder>\n  git checkout -- <file/folder>${RESET}"

--- a/Makefile
+++ b/Makefile
@@ -127,7 +127,7 @@ template_init:
 	@mv GETTING_STARTED .github/template_specific/ && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "\nCreating a clean README.md"
-	@cat .github/template_specific/README_template.md >> README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
+	cat .github/template_specific/README_template.md >> README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "Creating a clean CHANGELOG.md "
 	@echo "# Changelog\n\n## 0.0.1 [UNNRELEASED]\n\n###Added\n\n###Changed\n\n###Fixed\n" >> CHANGELOG.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"

--- a/Makefile
+++ b/Makefile
@@ -7,33 +7,18 @@ export PATH := $(shell pwd)/tmp:$(PATH)
 ###################################
 ########## Color Scheme ###########
 ###################################
-ifneq (,$(findstring xterm,${TERM}))
-	BLACK        := \033[0;30m
-	RED          := \033[0;31m
-	GREEN        := \033[0;32m
-	ORANGE       := \033[0;33m
-	BLUE         := \033[0;34m
-	PURPLE       := \033[0;35m
-	CYAN         := \033[0;36m
-	LIGHTGREY    := \033[0;37m
-	DARKGREY     := \033[1;30m
-	YELLOW       := \033[1;33m
-	WHITE        := \033[1;37m
-	RESET 		 := \033[0m
-else
-	BLACK        := ""
-	RED          := ""
-	GREEN        := ""
-	ORANGE       := ""
-	BLUE         := ""
-	PURPLE       := ""
-	CYAN         := ""
-	LIGHTGREY    := ""
-	DARKGREY     := ""
-	YELLOW       := ""
-	WHITE        := ""
-	RESET 		 := ""
-endif
+BLACK        := \033[0;30m
+RED          := \033[0;31m
+GREEN        := \033[0;32m
+ORANGE       := \033[0;33m
+BLUE         := \033[0;34m
+PURPLE       := \033[0;35m
+CYAN         := \033[0;36m
+LIGHTGREY    := \033[0;37m
+DARKGREY     := \033[1;30m
+YELLOW       := \033[1;33m
+WHITE        := \033[1;37m
+RESET        := \033[0m
 
 ###################################
 ########## Development ############

--- a/Makefile
+++ b/Makefile
@@ -130,7 +130,7 @@ template_init:
 	cat .github/template_specific/README_template.md >> README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "Creating a clean CHANGELOG.md "
-	@echo "# Changelog\n\n## 0.0.1 [UNNRELEASED]\n\n###Added\n\n###Changed\n\n###Fixed\n" >> CHANGELOG.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
+	echo "# Changelog\n\n## 0.0.1 [UNNRELEASED]\n\n###Added\n\n###Changed\n\n###Fixed\n" >> CHANGELOG.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo "${BLUE}\nDone! You are all set to start developing!${RESET}"
 	@echo "${YELLOW}\nPS! If you want to keep a deleted folder, you can undo by running:\n  git reset HEAD <file/folder>\n  git checkout -- <file/folder>${RESET}"

--- a/Makefile
+++ b/Makefile
@@ -8,25 +8,31 @@ export PATH := $(shell pwd)/tmp:$(PATH)
 ########## Color Scheme ###########
 ###################################
 ifneq (,$(findstring xterm,${TERM}))
-	BLACK        := $(shell tput -Txterm setaf 0)
-	RED          := $(shell tput -Txterm setaf 1)
-	GREEN        := $(shell tput -Txterm setaf 2)
-	YELLOW       := $(shell tput -Txterm setaf 3)
-	LIGHTPURPLE  := $(shell tput -Txterm setaf 4)
-	PURPLE       := $(shell tput -Txterm setaf 5)
-	BLUE         := $(shell tput -Txterm setaf 6)
-	WHITE        := $(shell tput -Txterm setaf 7)
-	RESET := $(shell tput -Txterm sgr0)
+	BLACK        := \033[0;30m
+	RED          := \033[0;31m
+	GREEN        := \033[0;32m
+	ORANGE       := \033[0;33m
+	BLUE         := \033[0;34m
+	PURPLE       := \033[0;35m
+	CYAN         := \033[0;36m
+	LIGHTGREY    := \033[0;37m
+	DARKGREY     := \033[1;30m
+	YELLOW       := \033[1;33m
+	WHITE        := \033[1;37m
+	RESET 		 := \033[0m
 else
 	BLACK        := ""
 	RED          := ""
 	GREEN        := ""
-	YELLOW       := ""
-	LIGHTPURPLE  := ""
-	PURPLE       := ""
+	ORANGE       := ""
 	BLUE         := ""
+	PURPLE       := ""
+	CYAN         := ""
+	LIGHTGREY    := ""
+	DARKGREY     := ""
+	YELLOW       := ""
 	WHITE        := ""
-	RESET        := ""
+	RESET 		 := ""
 endif
 
 ###################################

--- a/Makefile
+++ b/Makefile
@@ -123,7 +123,7 @@ template_init:
 	@echo -n "\nMoving README.md to .github/template_specific as old_README.md " 
 	mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo -n "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED"
+	@echo -n "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED "
 	mv GETTING_STARTED .github/template_specific/ && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo -n "\nCreating a clean README.md "

--- a/Makefile
+++ b/Makefile
@@ -126,7 +126,7 @@ template_init:
 	@echo -n "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED"
 	mv GETTING_STARTED .github/template_specific/ && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo -n "\nCreating a clean README.md"
+	@echo -n "\nCreating a clean README.md "
 	cat .github/template_specific/README_template.md >> README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
 	@echo -n "Creating a clean CHANGELOG.md "

--- a/Makefile
+++ b/Makefile
@@ -4,6 +4,30 @@ export PATH := $(shell pwd)/tmp:$(PATH)
 .ONESHELL .PHONY: up update-box destroy-box remove-tmp clean example
 .DEFAULT_GOAL := up
 
+###################################
+########## Color Scheme ###########
+###################################
+ifneq (,$(findstring xterm,${TERM}))
+	BLACK        := $(shell tput -Txterm setaf 0)
+	RED          := $(shell tput -Txterm setaf 1)
+	GREEN        := $(shell tput -Txterm setaf 2)
+	YELLOW       := $(shell tput -Txterm setaf 3)
+	LIGHTPURPLE  := $(shell tput -Txterm setaf 4)
+	PURPLE       := $(shell tput -Txterm setaf 5)
+	BLUE         := $(shell tput -Txterm setaf 6)
+	WHITE        := $(shell tput -Txterm setaf 7)
+	RESET := $(shell tput -Txterm sgr0)
+else
+	BLACK        := ""
+	RED          := ""
+	GREEN        := ""
+	YELLOW       := ""
+	LIGHTPURPLE  := ""
+	PURPLE       := ""
+	BLUE         := ""
+	WHITE        := ""
+	RESET        := ""
+endif
 
 ###################################
 ########## Development ############
@@ -90,28 +114,32 @@ else
 endif
 
 template_init:
-	@echo "\e[91m\nWarning! This will clean your template. Do you want to continue? [y/n]\e[0m" && read answer
-	if [ "$$answer" != "y" ]; then echo "Aborting!" && exit; fi
+	# 
+	@echo "${RED}\nWarning! This will clean your template. Do you want to continue? [y/n]${RESET}" ; \
+	read answer; \
+	if [ "$$answer" != "y" ]; then \
+		echo "Aborting!" ; \
+		exit 1 ; \
+	fi 
 
-	@echo "\nStarting to clean your template!\n\e[0m"
+	@echo "\nStarting to clean your template!" 
 
-	for folder in "template_example" "example/vagrant_box_example" "CHANGELOG.md"
-	do
-		echo -n "Deleting: $$folder "
-		rm -rf $$folder && echo "\e[32mSuccess\e[0m" || echo "\e[91mFailed\e[0m"
+	@for folder in "template_example" "example/vagrant_box_example" "CHANGELOG.md" ; do \
+		echo "Deleting: $$folder " ; \
+		rm -rf $$folder && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}" ; \
 	done
 
-	@echo -n "\nMoving README.md to .github/template_specific as old_README.md "
-	mv README.md .github/template_specific/old_README.md && echo "\e[32mSuccess\e[0m" || echo "\e[91mFailed\e[0m"
+	@echo "\nMoving README.md to .github/template_specific as old_README.md" 
+	@mv README.md .github/template_specific/old_README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo -n "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED "
-	mv GETTING_STARTED .github/template_specific/ && echo "\e[32mSuccess\e[0m" || echo "\e[91mFailed\e[0m"
+	@echo "Moving GETTING_STARTED/ to .github/template_specific/GETTING_STARTED"
+	@mv GETTING_STARTED .github/template_specific/ && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo -n "\nCreating a clean README.md "
-	cat .github/template_specific/README_template.md >> README.md && echo "\e[32mSuccess\e[0m" || echo "\e[91mFailed\e[0m"
+	@echo "\nCreating a clean README.md"
+	@cat .github/template_specific/README_template.md >> README.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo -n "Creating a clean CHANGELOG.md "
-	echo "# Changelog\n\n## 0.0.1 [UNNRELEASED]\n\n###Added\n\n###Changed\n\n###Fixed\n" >> CHANGELOG.md && echo "\e[32mSuccess\e[0m" || echo "\e[91mFailed\e[0m"
+	@echo "Creating a clean CHANGELOG.md "
+	@echo "# Changelog\n\n## 0.0.1 [UNNRELEASED]\n\n###Added\n\n###Changed\n\n###Fixed\n" >> CHANGELOG.md && echo "${GREEN}Success${RESET}" || echo "${RED}Failed${RESET}"
 
-	@echo "\e[32m\nDone! You are all set to start developing!\e[0m"
-	@echo "\e[93m\nPS! If you want to keep a deleted folder, you can undo by running:\n  git reset HEAD <file/folder>\n  git checkout -- <file/folder>\e[0m"
+	@echo "${BLUE}\nDone! You are all set to start developing!${RESET}"
+	@echo "${YELLOW}\nPS! If you want to keep a deleted folder, you can undo by running:\n  git reset HEAD <file/folder>\n  git checkout -- <file/folder>${RESET}"

